plyer/facades/orientation.py

```diff
@@ -41,5 +61,13 @@ def set_sensor(self, mode='any'):
         '''
         self._set_sensor(mode=mode)
 
+    # private
+
+    def _set_landscape(self, **kwargs):
```

Comment:
ditto,  this is a pr for document changes, do not do code changes in this.

Do separate pr's for code changes.